packages/shared/lib/constants.ts

```diff
@@ -1,2 +1,4 @@
 export const SYNC_TASK_QUEUE = 'nango-syncs';
 export const WEBHOOK_TASK_QUEUE = 'nango-webhooks';
+//export const CONNECTIONS_WITH_SCRIPTS_CAP_LIMIT = 3;
+export const CONNECTIONS_WITH_SCRIPTS_CAP_LIMIT = Infinity;
```

Comment:
is that on purpose?

Reply:
Yes, very much so. We don't want to cap until we can set the account information correctly via the API

Reply:
Just thought about it during the night (could not sleep). We also have to think about enterprise/community 👌🏻